installing/installing-fips.adoc

```diff
@@ -14,7 +14,7 @@ Because FIPS must be enabled before the operating system that your cluster uses
 [id="installation-about-fips-validation_{context}"]
 == FIPS validation in {product-title}
 
-{product-title} uses certain FIPS Validated / Modules in Process modules within {op-system-base} and {op-system} for the operating system components that it uses. See link:https://access.redhat.com/articles/3359851[RHEL7 core crypto components]. For example, when users SSH into {product-title} clusters and containers, those connections are properly encrypted.
+{product-title} uses certain FIPS Validated / Modules in Process modules within {op-system-base} and {op-system} for the operating system components that it uses. See link:https://access.redhat.com/articles/3655361[RHEL8 core crypto components]. For example, when users SSH into {product-title} clusters and containers, those connections are properly encrypted.
```

Comment:
This is true for 'core components' (running on RHCOS); however RHEL 7 workers are also supported on 4.1->4.8. For this reason; we need to likely elaborate more on how to find the FIPS packages for the OS your container runs on (and thus the correct KCS article).

Reply:
Yes, it is a little confusing here. 
@knewcomerRH, @sosiouxme  is that Okay to say "for RHCOS nodes, see link https://access.redhat.com/articles/3655361[RHEL8 core crypto components]; for Rhel7 nodes, see link:https://access.redhat.com/articles/3359851[RHEL7 core crypto components]"?  Any suggestion? Thanks.